### PR TITLE
Additional Exosuit Modules

### PR DESCRIPTION
**Pull request by Drew4484**
#### Summary
Content "Adds more exosuit modules."

#### Purpose of change
Adds some new modules expanding the utility options for an exosuit.
Climbing: Allows climbing walls as per the WALL_CLING effect.
Shield: Replicates the bash/cut & ballistic/pierce shield CBMs, requires toggling between the two states.
Jetpack: Allows a rapid leap across hostile terrain. 
Exosuit Air Supply Module: A small air tank that slowly transforms C02 to breathable air via an engineered bacteria.
Medical injector: RX12 injector with a larger capacity than the handheld version.
Structural Imager Module: As the handheld version, more compact as it runs entirely off of suit power.

#### Describe the solution
Modules

- [x] Climbing
- [x] Jetpack
- [x] Shield
- [x] Organic oxygen supply 
- [x] Medical support unit (like RX12)
- [x] Structural imager

#### Describe alternatives you've considered
None

#### Testing
WIP

#### Additional context
Replaces #56976 which had become extremely spaghetti'd due to Git and user issues.

### CODE REVIEW

**Comment by John-Candlebury** [comment]
Is this one ready for testing?
Just noticed your six tasks are marked as done but its still a draft

File: data/mods/Aftershock/items/armor/exosuit/exosuit_modules.json [hunk 1]
```diff
@@ -317,5 +317,227 @@
...
+      "target": "exo_forcefield_bash_cut",
+      "active": true,
+      "need_charges": 1,
+      "need_charges_msg": "There is not enough charge to activate the module."
+    }
+  },
+  {
+    "id": "exo_forcefield_bash_cut",
+    "type": "TOOL",
+    "name": { "str": "exosuit forcefield generator (bash & cut)", "str_pl": "exosuit forcefield generators (bash & cut) },
```
Comment (dseguin) [inline data/mods/Aftershock/items/armor/exosuit/exosuit_modules.json:464]:
```suggestion
    "name": { "str": "exosuit forcefield generator (bash & cut)", "str_pl": "exosuit forcefield generators (bash & cut)" },
```

File: data/mods/Aftershock/items/armor/exosuit/exosuit_modules.json [hunk 2]
```diff
@@ -317,5 +317,227 @@
...
+      "menu_text": "Turn to kinetic/pierce",
+      "type": "transform",
+      "msg": "You toggle the forcefield to protect against ballistic and pierce.",
+      "target": "exo_forcefield_ballistic_pierce"
+    }
+  },
+  {
+    "id": "exo_forcefield_ballistic_pierce",
+    "type": "TOOL",
+    "name": { "str": "exosuit forcefield generator (ballistic & pierce)", "str_pl": "exosuit forcefield generators (ballistic & pierce)" },
```
Comment (dseguin) [inline data/mods/Aftershock/items/armor/exosuit/exosuit_modules.json:497]:
```suggestion
    "name": {
      "str": "exosuit forcefield generator (ballistic & pierce)",
      "str_pl": "exosuit forcefield generators (ballistic & pierce)"
    },
```

File: data/mods/Aftershock/items/armor/exosuit/exosuit_modules.json [hunk 3]
```diff
@@ -317,5 +317,230 @@
...
+    "looks_like": "storage_battery",
+    "color": "light_gray",
+    "armor": [ { "encumbrance": 5, "covers": [ "arm_l", "arm_r", "hand_l", "hand_r", "leg_l", "leg_r", "foot_l", "foot_r" ] } ],
+    "flags": [ "CANT_WEAR", "EXO_MEDIUM_GADGET" ]
+  },
+  {
+    "id": "exo_jetpack",
+    "type": "TOOL",
+    "name": { "str": "exosuit dynamic maneuvering unit" },
+    "description": "A large collection of tanks and nozzles, this module converts hydrogen to plasma, propelling the user between spaceships or across rough terrain and gaps.  The thrust is insufficent for flight in a gravity well, but can still launch you forward at extreme speed.",
```
Comment (dseguin) [inline data/mods/Aftershock/items/armor/exosuit/exosuit_modules.json:415]:
```suggestion
    "description": "A large collection of tanks and nozzles, this module converts hydrogen to plasma, propelling the user between spaceships or across rough terrain and gaps.  The thrust is insufficient for flight in a gravity well, but can still launch you forward at extreme speed.",
```

**Pull request merged**